.github/workflows/macos-build.yml

```diff
@@ -326,6 +361,91 @@ jobs:
         echo 'RBE_service='`node -e "console.log(require('./src/utils/reclient.js').serviceAddress)"` >> $GITHUB_ENV
         echo 'RBE_experimental_credentials_helper='`node -e "console.log(require('./src/utils/reclient.js').helperPath({}))"` >> $GITHUB_ENV
         echo 'RBE_experimental_credentials_helper_args=print' >> $GITHUB_ENV
+    - name: Free Space on MacOS
```

Comment:
Did we decide to do this pre-emptively or are we already running into the same issues?

Reply:
Unfortunately we did end up needing this for Intel (`macos-13-large`) 😕 I'll see if this is still needed for `macos-14-large`

Reply:
let's just leave it for now imo!